lxd/device/disk.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/lxc/lxd/lxd/instance/instancetype"
```

Comment:
Typo in commit, should be in `Store the storage pool`

Reply:
Thanks! fixed.